.github/workflows/publish.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Rust SDK
+on: [push, pull_request]
+jobs:
+  google-cloud-auth:
```

Comment:
The name should be something like `packaging-dry-run` that is neutral w.r.t. to the package being tested.

Reply:
I was thinking of simply calling it `packages` .

```yaml
jobs:
  packages:
    runs-on: ubuntu-24.04
    strategy:
      matrix:
        packages: ['auth', 'wkt', 'gax']
```

This would result to a ci title on push for the package wkt to:  `Publish Rust SDK / packages (wkt) (push)`.